### PR TITLE
Billing RP spec changes for listInvoiceSectionsWithCreateSubscription…

### PR DESCRIPTION
**Pull request by asarkar84**
…Permission API

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by wilcobmsft — APPROVED** [review]

**Review by msft-adrianma — APPROVED** [review]

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 1]
```diff
@@ -158,6 +158,42 @@
...
+            "schema": {
+              "$ref": "#/definitions/InvoiceSectionListWithCreateSubPermissionResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: missing x-ms-pageable

**Review by wilcobmsft — CHANGES_REQUESTED** [review]

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json (hunk 1 above)
Comment (asarkar84) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: Sure! Added the property.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 2]
```diff
@@ -5529,6 +5526,51 @@
...
+    "InvoiceSectionListWithCreateSubPermissionResult": {
+      "type": "object",
+      "description": "Result of listing invoice section properties with create subscription permission.",
+      "properties": {
+        "value": {
+          "description": "The list of invoice section properties with create subscription permission.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InvoiceSectionWithCreateSubPermission"
+          }
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: missing:
        "nextLink": {
          "description": "The link (url) to the next page of results.",
          "type": "string",
          "readOnly": true
        }

Comment (asarkar84) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: Sure, Added.

**Review by wilcobmsft — APPROVED** [review]

**Review by mmyyrroonn — APPROVED** [review]
LGTM. This PR contains BreakingChanges. It's preview version.

**Comment by MyronFanQiu** [comment]
@KrisBash @asarkar84 Hello. Do you discuss about this PR offline? @asarkar84 Do you know how to change this PR?

**Comment by asarkar84** [comment]
@MyronFanQiu I have not discussed the changes with @KrisBash offline.
@KrisBash I am not able to see the comments from you in this PR. Please add them.

**Review by KrisBash — COMMENTED** [review]
> @KrisBash I am not able to see the comments from you in this PR. Please add them.

@asarkar84  I'm not sure what the issue is -  I see my comments directly above this and in the PR code diff.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 3]
```diff
@@ -2630,48 +2669,6 @@
         }
       }
     },
-    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/products/{productName}/updateAutoRenew": {
```
[the remark this answers is not in the document]
Comment (asarkar84) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: This API does not exist, this was a swagger bug.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 4]
```diff
@@ -158,6 +158,45 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/listInvoiceSectionsWithCreateSubscriptionPermission": {
```
[the remark this answers is not in the document]
Comment (asarkar84) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: You are right. This is a post API. Making the changes.

**Comment by asarkar84** [comment]
@KrisBash I have incorporated the review comments. Kindly take a look.

**Comment by MyronFanQiu** [comment]
@KrisBash Truly sorry! I just made a mistake that not wait for your confirmation. Could you please check whether the author has solved your comments. @asarkar84 Hello. I'm sorry I just merged this PR. Just as a reminder, we may need further changes. Truly sorry for my mistake.

**Pull request merged**